spring-security/src/test/java/com/sap/cloud/security/spring/token/authentication/ReactiveHybridJwtDecoderTest.java

```diff
@@ -46,6 +46,18 @@ void parseJwt() {
 
 	}
 
+	@Test
+	void parseJwtXsuaaToken() {
+		JwtGenerator jwtGeneratorXsuaa = JwtGenerator.getInstance(XSUAA, "theClientId").withClaimValue("iat", "1704063600");
+
+		Jwt jwtToken = ReactiveHybridJwtDecoder.parseJwt(jwtGeneratorXsuaa.createToken()).block();
+
+		assertEquals(3, jwtToken.getHeaders().size());
+		assertEquals(7, jwtToken.getClaims().size());
+		assertEquals(1, jwtToken.getExpiresAt().compareTo(Instant.now()));
```

Comment:
I guess you want to check here the issuedAt (iat) not the expiresAt claim (ext) 🤔